packages/kbn-lens-icons/src/assets/annotation_icons/circle.tsx

```diff
@@ -1,8 +1,9 @@
 /*
  * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
  * or more contributor license agreements. Licensed under the Elastic License
- * 2.0; you may not use this file except in compliance with the Elastic License
- * 2.0.
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
  */
```

Comment:
Do we want to change the license of these icons? If we put this package in `x-pack/packages` or somewhere else in the `x-pack` dir we can maintain the existing license.

Reply:
@spalger otherwise i see ts error

<img width="916" alt="image" src="https://user-images.githubusercontent.com/20072247/182904956-9cadd98a-6349-4e34-ba3d-8d73960843e9.png">

Reply:
Yes, if the package is in the root `packages` directory it has a one license, if it's in the `x-pack` directory it has a different license

Reply:
right, so if we decide to put it in a separate package, I don't think we have a choice with the license.  @flash1293  please have a look ^^^^^

Reply:
@alexwizp you can put packages anywhere, not just the `packages` directory https://docs.elastic.dev/kibana-dev-docs/ops/packages#where-do-i-put-my-package.

Reply:
@spalger thanks for that doc, didn't know about that

Reply:
It's brand new, I'm glad it's useful 🙂